### ISSUE TITLE
Support for environment variables?

### ISSUE THREAD

**Issue by brasilikum** [alexflint/go-arg#2]
I was wondering if parsing of environment variables is something worth adding? I thought of some method like `AutomaticEnv` in [viper](https://github.com/spf13/viper), where you can set an optional prefix and then parse Environment variables just like parameters. Is there interest in a push request?

**Comment by alexflint** [comment]
Yes this does seem worth supporting. Perhaps there should be a struct tag like
```
var args struct {
    RemoteHost string `env:"REMOTE_HOST"`
}
arg.MustParse(&args)
```

On the other hand, it would be nice to avoid typing the name of the argument twice. Will have to think more about this.

**Comment by brettlangdon** [comment]
This was resolved by #9 (which is now merged)
